### DISCUSSION TITLE
[SOLVED] so-desktop-install on Standalone install fails

### DISCUSSION THREAD

**Discussion by stummasino-matc** [Security-Onion-Solutions/securityonion#13064] [General] [answered]
### Issue:
As the title says, running so-desktop-install fails after doing a Standalone install of SO 2.4.60. This was working in my previous install (2.4.10). I have the **/opt/so/log/salt/minion** log attached and cleared it right before running so-desktop-install. It looks like it started going wrong right around line 18. Maybe a package version issue? 

### Log:
[minion.txt](https://github.com/Security-Onion-Solutions/securityonion/files/15409724/minion.txt)

### Problem from logs:
- Problem 1: package chromium-123.0.6312.105-1.el9.x86_64 from securityonion requires libQt5Core.so.5()(64bit), but none of the providers can be installed
- Problem 2: package mono-devel-6.12.0-5.el9.x86_64 from securityonion requires glib2-devel, but none of the providers can be installed
- Problem 3: package wireshark-1:3.4.10-6.el9.x86_64 from securityonion requires libQt5Core.so.5()(64bit), but none of the providers can be installed

### System Info:
Security Onion version:**2.4.60**
Is this a cloud deployment or on-prem?: **on-prem**
Internet access or this an airgap installation?: **Internet**
Did you install from our Security Onion ISO image or did you perform a network installation?: **ISO**
How many nodes do you have?: **Standalone**
What are the hardware specs of each of those nodes?: **Double the mins on all (Not heavey traffic)**
Are you experiencing issues monitoring network traffic?: **No**
Does so-status show all services running?: **Yes**
Do you get any failures when you run sudo salt-call state.highstate?: **No**
Does the SOC Grid page show any failures? **No**

**Comment by m0duspwnens** [comment, marked as answer]
We have found the issue and resolved it. On your manager, as the socore user, you can run `so-repo-sync`. This will download the new packages. From there, you should be able to run `salt-call state.highstate` on the standalone node and the desktop install will complete.

**Comment by stummasino-matc** [comment]
That fixed it!!! Thanks
